CHANGELOG.md

```diff
@@ -12,6 +12,8 @@
 
 ### Added
 
+- Add `EndLedger` in `GetEventsResponse`. This tells the client until what ledger events are being queried. e.g.: `startLEdger` (inclusive) - `endLedger` (exclusive)
```

Comment:
This needs to be updated since it's `cursor` now

Reply:
oh yes !! Done now :)